### PR TITLE
Completed array-map - Aaron Butnik

### PR DESCRIPTION
**Pull request by aaron-j-butnik**
image

### CODE REVIEW

File: array-map/main.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  'Structured Query Language'
+];
+
+// All numbers multiplied by 2.
+const doubled = numbers => numbers.map(element => element * 2);
+console.log(doubled(numbers));
+
+// All numbers formatted into "$0.00" strings.
+const prices = numbers => numbers.map(element => `$${element}.00`);
+console.log(prices(numbers));
```
Comment (TimelikeClosure) [inline array-map/main.js]: Please make sure this produces valid price strings, even if the numbers already have their own decimal places (ex: `[5.2, 13.82365]` should turn into `["$5.20", "$13.82"]`).

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
Please make sure `prices` produces valid price strings, even if the numbers already have their own decimal places (ex: `[5.2, 13.82365]` should turn into `["$5.20", "$13.82"]`).

**Comment by aaron-j-butnik** [comment]
image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
Please make sure `prices` produces valid price strings, even if the numbers already have their own decimal places (ex: `[5.2, 13.82365]` should turn into `["$5.20", "$13.82"]`).

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**